pkgs/os-specific/linux/dpdk/default.nix

```diff
@@ -68,7 +68,7 @@ in stdenv.mkDerivation rec {
   ]
   # kni kernel driver is currently not compatble with 5.11
   ++ lib.optional (mod && kernel.kernelOlder "5.11") "-Ddisable_drivers=kni"
-  ++ lib.optional (!shared) "-Ddefault_library=static"
+  ++ [(if shared then "-Ddefault_library=shared" else "-Ddefault_library=static")]
```

Comment:
I wonder if we should stick to shared by default?

Reply:
Dpdk's intention seems to be development related (https://github.com/dpdK/dpdk/commit/93b1f90ae17425c496257bacb21d9699dd072cf3). I'd say we should be able to default to shared.